### PR TITLE
[feat](Nereids) support alter job status commands

### PR DESCRIPTION
**Pull request by LiBinfeng-01**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 
    - [x] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by LiBinfeng-01** [comment]
run buildall

**Comment by LiBinfeng-01** [comment]
run buildall

**Comment by LiBinfeng-01** [comment]
run external

File: fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java [hunk 1]
```diff
@@ -4127,6 +4168,17 @@ public LogicalPlan visitShowVariables(ShowVariablesContext ctx) {
         }
     }
 
+    private Expression getWildWhere(DorisParser.WildWhereContext ctx) {
+        if (ctx.LIKE() != null) {
+            String pattern = ctx.STRING_LITERAL().getText();
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java:4173]:
```suggestion
            String pattern = stripQuotes(ctx.STRING_LITERAL().getText());
```

**Comment by LiBinfeng-01** [comment]
run buildall

**Comment by LiBinfeng-01** [comment]
run performance

**Review by starocean999 — APPROVED** [review]

**Review by echo-hhj — APPROVED** [review]
LGTM

**Pull request merged**